clients/vault/src/oracle/agent.rs

```diff
@@ -210,7 +210,7 @@ impl OracleAgent {
 #[cfg(test)]
 mod tests {
 	use crate::oracle::{
-		get_random_secret_key, get_test_secret_key, specific_stellar_relay_config,
+		get_mainnet_secret_key, get_random_secret_key, specific_stellar_relay_config,
```

Comment:
addresses https://github.com/pendulum-chain/spacewalk/blob/bda384bcc6db3da88198c714851a261dcdea1063/clients/vault/tests/helper/mod.rs#L33.
We have 2 testnet secret keys. To cleanly switch between testing at mainnet or testnet, separeted what was once just `fn get_test_secret_key()` to:
* `get_mainnet_secret_key()`
* `get_testnet_secret_key(...)`